sky/skylet/autostop_lib.py

```diff
@@ -121,4 +122,5 @@ def is_autostopping(cls) -> str:
     def _build(cls, code: List[str]) -> str:
         code = cls._PREFIX + code
         code = ';'.join(code)
-        return f'python3 -u -c {shlex.quote(code)}'
+        return (f'{constants.ACTIVATE_PYTHON_ENV} '
```

Comment:
actually, why not use `sky/skylet/constants.py::run_in_python_env` to replace all of those `constants.ACTIVATE_PYTHON_ENV`?